Waidhoferj/CSC-308-309-Project
# Added basic profile page

Added a profile page that displays use name, image, metrics and bio. If there is a fetch error, will display error message with "go back" option.

## Limitations
- It always fetches the 2nd user in the server (to successfully display metrics)
- The profile pic is filler

## Testing
No connection
- Run the frontend dev server
- Go to `/profile`
- Ensure that error message is displayed
With server
- Then start up backend server
- Go to `/profile` in the browser
- Ensure that all information displays
